### ISSUE TITLE
GitHub backed ignores prefix.

### ISSUE THREAD

**Issue by voxik** [fedora-infra/anitya#372]
I just received several notification:

```
A new version of "Vagrant" has been detected: "v1.8.7" newer than "1.8.6", packaged as "vagrant"
A new version of "fugitive.vim" has been detected:  "v2.2", packaged as "vim-fugitive"
A new version of "commentary.vim" has been detected:  "v1.3", packaged as "vim-commentary"
```

All these repositories has set "v" version prefix, so it should be ignored, but it is not. This started to happen approximately one week ago.

**Comment by akien-mga** [comment]
Confirmed.

**Comment by akien-mga** [comment]
Here's an example of a project where the version prefix was ignored: https://release-monitoring.org/project/12312/

**Comment by akien-mga** [comment]
It's a priori a regression in 0.10, as I don't remember this happening previously.

**Comment by pypingou** [comment]
> It's a priori a regression

What is the difference between a regression and a bug? :)

**Comment by akien-mga** [comment]
Well it's a bug that did not happen in a known previous version, which allows e.g. to bisect between those two versions to find out what commit created the bug :)

**Comment by pypingou** [comment]
Ah, so it is a bug :)

/me just teasing here

**Comment by jeremycline** [comment]
I can't manage to reproduce this in my dev environment. The leading ``v`` seems to be stripped correctly in the test cases I've managed to come up with. I think maybe https://github.com/fedora-infra/anitya/pull/305 or https://github.com/fedora-infra/anitya/pull/362 introduced the regression, based on their commit messages, but I don't really know. I also don't know if https://github.com/fedora-infra/anitya/pull/370 fixes it or even what prompted that change. @pypingou do you have any insights here?

**Comment by pypingou** [comment]
@jeremycline not from the top of my head, I've been meaning to look closer into it but haven't managed yet :(

**Comment by pypingou** [comment]
I've been trying to reproduce the issue using vagrant, cutegram, plexus-archiver and sc-controller, all are behaving as expected locally.

I wonder if this could be related to the cron job, but since it's using the same code path, I don't see how it would be different :(

**Comment by voxik** [comment]
Today I got two notifications:

```
A new version of "Vagrant" has been detected:  "1.8.7", packaged as "vagrant"
```

and 10 minutes later:

```
A new version of "Vagrant" has been detected:  "v1.8.7", packaged as "vagrant"
```

**Comment by jeremycline** [comment]
I think I finally figured out the problem as I went through the changes to make a changelog. Prior to https://github.com/fedora-infra/anitya/commit/1de2b6817eb6ebcc34b6ca062904911c75db4919 the prefix was stripped, but not reinserted into the list. This _should_ be fixed in the upcoming 0.10.1 release.
